docs/guide/cache/content_aware_cache.md

```diff
@@ -293,3 +350,253 @@ bin/console fos:httpcache:invalidate:tag ez-all
+located on that cluster (`production` and sometimes also `staging`) is slightly different:
+In the `URLs` dropdown in the Platform.sh dashboard you need to find the route that has the format 
+`something.[clusterid].ent.platform.sh/`, for instance `myenvironment.asddfs2323.ent.platform.sh/`
+
+The endpoint will in that case be in the format `c.[clusterid].ent.platform.sh`, for instance `c.asddfs2323.ent.platform.sh/`
+Next, use nslookup to find the IP:
+
+```bash
+    $ nslookup
+    > c.asddfs2323.ent.platform.sh
+   (...)
+   Address:  1.2.3.4
+```
+
+### Fetching the User-Context-Hash
+
+As explained in [User Context Hash caching](#user-context-hash-caching), the HTTP Cache indexes the cache based on the
+user-context-hash. Users with the same user-context-hash will share the same cache (as long as [[= product_name =]]
+responds with `Vary: X-User-Hash`).
+
+So in order to simulate the requests the HTTP-Cache sends to [[= product_name =]], we need that user-context-hash.
+That can be obtained using `curl`
+
+```bash
+    $ curl -IXGET --resolve www.staging.foobar.com.us-2.platformsh.site:443:1.2.3.4 --header "Surrogate-Capability: abc=ESI/1.0" --header "accept: application/vnd.fos.user-context-hash" --header "x-fos-original-url: /" https://www.staging.foobar.com.us-2.platformsh.site/_fos_user_context_hash
+```
+
+Some notes about each of these parameters:
+- `-IXGET`, one of many ways to tell curl that we want to send a GET request, but we are only interested in outputting the headers
+- `--resolve www.staging.foobar.com.us-2.platformsh.site:443:1.2.3.4`
+  - We tell curl not to do a DNS lookup for `www.staging.foobar.com.us-2.platformsh.site`. We do that because in our case
+    that will resolve to the Fastly endpoint, not our origin (nginx)
+  - We specify `443` because we are using `https`
+  - We provide the IP of the nginx endpoint at platform.sh (`1.2.3.4` in this example)
+- `--header "Surrogate-Capability: abc=ESI/1.0"`, strictly speaking not needed when fetching the user-context-hash, but this tells [[= product_name =]] that client understands ESI tags.
+  It is good practice to always include this header when imitating the HTTP Cache.
+- `--header "accept: application/vnd.fos.user-context-hash"` tells [[= product_name =]] that the client wants to receive the user-context-hash
+- `--header "x-fos-original-url: /"` is required by the fos-http-cache bundle in order to deliver the user-context-hash
+- `https://your-page-blah-blah.us-2.platformsh.site/_fos_user_context_hash` : here we use the hostname we earlier told
+  curl how to resolve using `---resolve`. `/_fos_user_context_hash` is the route to the controller that are able to
+  deliver the user-context-hash.
+- You may also provide the session cookie (`--cookie ".....=....") for a logged-in-user if you are interested in
+  the x-user-context-hash for a different user but anonymous
+
```

Comment:
```suggestion
Parameters description:

- `-IXGET`, one methods to tell curl that you want to send a GET request, but you are interested in outputting the headers

- `--resolve www.staging.foobar.com.us-2.platformsh.site:443:1.2.3.4`
  - you tell curl not to do a DNS lookup for `www.staging.foobar.com.us-2.platformsh.site`. It is done because in this case
    it resolves to the Fastly endpoint, not the origin (Nginx)
  - You specify `443` because you use `https`
  - You provide the IP of the Nginx endpoint at Platform.sh (`1.2.3.4` in this example)
  
- `--header "Surrogate-Capability: abc=ESI/1.0"`, not needed when fetching the user-context-hash, but it tells [[= product_name =]] that client understands ESI tags.
  It is good practice to always include this header when imitating the HTTP cache.
  
- `--header "accept: application/vnd.fos.user-context-hash"` tells [[= product_name =]] that the client wants to receive the user-context-hash

- `--header "x-fos-original-url: /"` is required by the fos-http-cache bundle in order to deliver the user-context-hash

- `https://your-page-blah-blah.us-2.platformsh.site/_fos_user_context_hash` - it uses the hostname that was earlier told
  curl how to resolve using `---resolve`. `/_fos_user_context_hash` is the route to the controller that is able to
  deliver the user-context-hash.
  
- You can also provide the session cookie (`--cookie ".....=....") for a logged-in-user if you are interested in
  the x-user-context-hash for a different user but anonymous.

```